### ISSUE TITLE
Phone number box is taking alphabetic value and number exceeds 12 digits without country code also

### ISSUE THREAD

**Issue by Mayanksaininh** [ANSHIKA-26/WordWise#1103]
**Describe the bug**
Phone number box is taking alphabetic value and number exceeds 12 digits without country code also

**To Reproduce**
1. Go to -the website
2. Click on -Contact us
3. scroll down to phone number box.

**Expected behavior**
There should be limit number will not exceeds above 12 digits and does not take any type of alphabetic value.

**Screenshots**
![Screenshot (47)](https://github.com/user-attachments/assets/d3bed648-3ca4-4216-a18d-db519b6db356)
![Screenshot (48)](https://github.com/user-attachments/assets/62a9b757-dffb-4302-8ffb-64b86c0aeec3)

**Additional context**

**Comment by Mayanksaininh** [comment]
Please assign this issue to me

**Comment by ANSHIKA-26** [comment]
why did u not fix it earlier ?

**Comment by ANSHIKA-26** [comment]
were u not assigned something related to this ? in your old PR you added the phone number box right ?

**Comment by Mayanksaininh** [comment]
I resolve this 3 times but I don't know what is the problem

**Comment by ANSHIKA-26** [comment]
and now you know how  to fix this ? thats why you have raised this issue ?

**Comment by Mayanksaininh** [comment]
Yes I know how to fix, that time I also fixed, my code was ran in my machine. 
But after PR generation it show me conflict

**Comment by haseebzaki-07** [comment]
hey @ANSHIKA-26 Can I take this issue?? I will create the PR soon!

**Comment by ANSHIKA-26** [comment]
@Mayanksaininh ??

**Comment by NK-Works** [comment]
@ANSHIKA-26 i can wok on it if you want
